homework/index.js

```diff
@@ -0,0 +1,24 @@
+const inputValue = document.querySelector('#search');
+const searchButton = document.querySelector('.searchButton');
+const reposContainer = document.querySelector('.main__profile-repos');
+// const clientId = '1ebd35cdc9277ed1ee8c';
+// const clientSecret = '0ecdc67b56e4ca7c3b4710b60f135d52a8f11a32';
+const fetchRepo = async () => {
+  // const apiCall = await fetch(`https://api.github.com/users/${user}?clientId=${clientId}&clientSecret=${clientSecret}`);
+  const apiCall = await fetch('https://api.github.com/HackYourFuture/repos?callback=?');
```

Comment:
Looks like you've changed the endpoint url here. Did you maybe do that to test the error flow? The correct url is in the homework guide (or here: https://api.github.com/orgs/HackYourFuture/repos?per_page=100 ) ;)